tests/bsim/bluetooth/audio/src/bap_broadcast_source_test.c

```diff
@@ -373,7 +361,7 @@ static int setup_extended_adv(struct bt_bap_broadcast_source *source, struct bt_
 		return err;
 	}
 
-	test_broadcast_source_get_id(source, &broadcast_id);
+	broadcast_id = 0x123456;
```

Comment:
Why a static ID here, when we use random in nearly all other cases?

Reply:
Its a miss :) I will update